docs/csharp/language-reference/keywords/sealed.md

```diff
@@ -27,6 +27,12 @@ In the following example, `Z` inherits from `Y` but `Z` cannot override the virt
 [!code-csharp[csrefKeywordsModifiers#16](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csrefKeywordsModifiers/CS/csrefKeywordsModifiers.cs#16)]
 
 When you define new methods or properties in a class, you can prevent deriving classes from overriding them by not declaring them as [virtual](virtual.md).
+When you override method inherited from <xref:System.Object?displayProperty=nameWithType> class, which are already virtual, you can prevent deriving classes from overriding them by using sealed keyword as in example:
+(feature [introduced with C# 10](../../whats-new/csharp-version-history.md))
```

Comment:
I'd like to suggest a different change to the docs here. 

You're right that we should mention that you can add the `sealed` modifier to prevent further derived types from adding an additional override. However, that feature has been part of C# since version 1.0. 

What was added in C# 10 was that the in C# 9, a hand-written `ToString` in a `record` type was explicitly prohibited from adding the `sealed` modifier, unless the `record` type was `sealed`.

Emphasizing that bit of evolution doesn't help. Instead, can we keep the example, and change the text as follows:

```suggestion
When you override a `virtual` member declared in a base type, you can prevent deriving types from overriding them by using sealed keyword as in the following example:
```